src/main/java/executor/command/CommandGetMonthlySpending.java

```diff
@@ -0,0 +1,76 @@
+package executor.command;
+
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.ReceiptTracker;
+import ui.Wallet;
+
+public class CommandGetMonthlySpending extends Command {
```

Comment:
Consider refactoring the class to _CommandGetSpendingByMonth_ or _CommandGetExpenditureByMonth_ for greater naming clarity.